src/base/bittorrent/session.cpp

```diff
@@ -3068,6 +3070,8 @@ void Session::startUpTorrents()
         if (!addTorrent_impl(params.addTorrentData, params.magnetUri, TorrentInfo::loadFromFile(filePath), params.data))
             logger->addMessage(tr("Unable to resume torrent '%1'.", "e.g: Unable to resume torrent 'hash'.")
                                .arg(params.hash), Log::CRITICAL);
+        // don't waste memory with to many unprocessed queue entries
+        readAlerts();
```

Comment:
@sledgehammer999 I don't know when I start to do this, maybe on the weekend, just to inform you.